KsApi/models/GraphUserCreditCard.swift

```diff
@@ -24,17 +24,22 @@ public struct GraphUserCreditCard: Swift.Decodable {
 
   public enum CreditCardType: String, Decodable, CaseIterable {
     case amex = "AMEX"
+    case diners = "DINERS"
     case discover = "DISCOVER"
     case jcb = "JCB"
     case mastercard = "MASTERCARD"
+    case unionPay = "UNION_PAY"
     case visa = "VISA"
-    case diners = "DINERS"
     case generic = "----"
 
     public var description: String? {
       switch self {
       case .amex, .discover, .jcb, .mastercard, .visa, .diners:
         return self.rawValue.capitalized
+      case .unionPay:
+        return self.rawValue
```

Comment:
Do you think it would make sense to introduce a helper function on a String object something like

```
extension String {
  func snakeToCamel() {
    return self.replacingOccurences(of: "_", with: " ")
  }
}
```

which could be testable..so that then we can simply use `return self.rawValue.snakeToCamel().capitalized`?

Reply:
I think that for now, since that's the only place where this transformation needs to be made, we don't need to extract the code to a helper function. It's a good idea though, in case we use the same logic somewhere else.